### ISSUE TITLE
how to define the batch size

### ISSUE THREAD

**Issue by Tangzy7** [yhenon/keras-frcnn#36]
Thanks to your codes!
I saw you used train_on_batch to start training. But how to define the batch size if you used that? I just guess that the batch size is always 1.

**Comment by yhenon** [comment]
This repo follows the original faster-rcnn paper, see section 3.1:

> The   RPN   can   be   trained   end-to-end   by   back-
propagation  and  stochastic  gradient  descent  (SGD)
[35]. We follow the “image-centric” sampling strategy
from [2] to train this network. Each mini-batch arises
from a single image that contains many positive and
negative  example  anchors

It could certainly be modified to do larger batches. You'll have to make sure to crop the images to all have equal size though, then concatenate them into a batch.
